tests/test_deepedit_transforms.py

```diff
@@ -14,81 +14,303 @@
 import numpy as np
 from parameterized import parameterized
 
-from monai.apps.deepedit.transforms import ClickRatioAddRandomGuidanced, DiscardAddGuidanced, ResizeGuidanceCustomd
+from monai.apps.deepedit.transforms import (
+    AddGuidanceFromPointsCustomd,
+    AddGuidanceSignalCustomd,
+    AddInitialSeedPointMissingLabelsd,
+    AddRandomGuidanceCustomd,
+    DiscardAddGuidanced,
+    FindAllValidSlicesMissingLabelsd,
+    FindDiscrepancyRegionsCustomd,
+    NormalizeLabelsInDatasetd,
+    ResizeGuidanceMultipleLabelCustomd,
+    SingleLabelSelectiond,
+    SplitPredsLabeld,
+    ToCheckTransformd,
+)
+from monai.utils import min_version, optional_import
 from monai.utils.enums import PostFix
 
-IMAGE = np.array([[[[1, 0, 2, 0, 1], [0, 1, 2, 1, 0], [2, 2, 3, 2, 2], [0, 1, 2, 1, 0], [1, 0, 2, 0, 1]]]])
-LABEL = np.array([[[[0, 0, 0, 0, 0], [0, 1, 0, 1, 0], [0, 0, 1, 0, 0], [0, 1, 0, 1, 0], [0, 0, 0, 0, 0]]]])
+measure, _ = optional_import("skimage.measure", "0.14.2", min_version)
+
+np.random.seed(0)
```

Comment:
please use the setup/teardown API to set the seeds, see https://github.com/Project-MONAI/MONAI/blob/c51d85e7a878ba0a63305fe89e0165f6463d55a2/tests/test_grid_dataset.py#L29-L33

then the test data would be created within the test cases.

Reply:
Please check the updated version